src/main/java/org/folio/dataexp/service/permissions/RequiredPermissionResolver.java

```diff
@@ -0,0 +1,11 @@
+package org.folio.dataexp.service.permissions;
+
+import org.springframework.stereotype.Component;
+
+@Component
+public class RequiredPermissionResolver {
+
+  public String getReadPermission() {
+    return "ui-inventory.instance.view";
```

Comment:
It is better to use constant to store permission.

Reply:
Used constant.